content/tutorials/max-gas-pub-data/10.index.md

```diff
@@ -0,0 +1,421 @@
+# Understanding Pubdata Costs in the ZKsync Era
```

Comment:
I might change the title to: "Pubdata Costs in ZKSync Era: An Essential Guide for Developers"